contrib/macdeploy/extract-osx-sdk.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+7z x Xcode_7.3.1.dmg 5.hfs
+SDKNAME="MacOSX10.11.sdk"
+SDKDIRINODE=154283
```

Comment:
Is it possible to specify this by name rather than by inode? That would make the script more future proof.